### PR TITLE
Add some basic tests for the notebook interface

### PR DESCRIPTION
**Pull request by takluyver**
This exercises some of the basic features of the notebook widgets interface. It's a starting point for more thorough tests.

### CODE REVIEW

File: geoAssembler/widgets/nb_tabs.py [hunk 1]
```diff
@@ -195,13 +194,13 @@ def _move_quadrants(self, pos):
         d = pos['owner'].description.lower()[0]
         pn = int(pos['new'])
         po = int(pos['old'])
-        sign = np.sign(pn - po)
+        delta = pn - po
```
Comment (antarcticrainforest) [inline geoAssembler/widgets/nb_tabs.py]: wouldn't it be more save to keep the signum here?

Comment (takluyver) [reply geoAssembler/widgets/nb_tabs.py]: I think that if you type a number in a text box, you can make the x or y pos change by more than 1px at a time. If you take the sign, then a change will only ever move it 1px, so the display position will get out of sync with the text boxes.

**Comment by antarcticrainforest** [comment]
I just wanted to look at this again. It seems the geometry is borken. At the the moving the quadrants doesn't work because it complains the the tile shapes to not match. I took a look into the release notes but I did not see any changes related to geometry.

**Comment by takluyver** [comment]
Maybe something changed in karabo_data that broke it. I'll see if I can figure it out.

**Comment by antarcticrainforest** [comment]
I think this was due to quadrant movement based on pixels rather than meters. If I am correct then karabo-data uses meters by default now? @takluyver  can you check if the changes make sense?
Also the tests are failing. I'll look into this!

**Comment by takluyver** [comment]
Thanks @antarcticrainforest. Yep, karabo_data (& now [EXtra-geom](https://github.com/European-XFEL/EXtra-geom), the new name for the geometry code) did switch to using metres rather than pixel units. Pixel units are really confusing when you get hexagonal pixels. :-)

**Pull request merged**